### PR TITLE
Replace hard-coded text, colors, fonts and measurements

### LINKED ISSUE #552: BUG: Create variables for hard coded text in the facility form and use theme for hard coded colors
# Problem
The facility form contains two major bugs that needs to be solved:
- The use of hard coded colors
- The use of hard coded text

# Success

Look for any hard coded colors
ex. border-zinc-400 -> border-secondary or border-primary-text-muted.
ex. bg-white, text-white, etc.
if time permits, try to shift hard coded font sizes, font families, etc. to tailwind style ex font-['Noto Sans JP'] can be font-sans

Create variables for hard coded text, placeholder text and error messages should have there own variables.

### Requirements

- [ ] Use our color theme for colors
- [ ] Create variables for all hard coded text

### PR DESCRIPTION
**Pull request by Anissa3005**
Resolves #552 

## 🔧 What changed

- I have replaced the hard-coded text with variables.
- I have replaced some hard-coded tailwind styles with our themes.
- I have replaced the px widths and heights using tailwind classes.

## 🧪 Testing instructions

For this PR there are no specific tests.

### CODE REVIEW

**Review by ermish — APPROVED** [review]
woo! looks much better 💯

**Comment by NabbeunNabi** [comment]
https://www.loom.com/share/abe38e4494bc48d689eed756a5001593 IT WORKS!

**Comment by NabbeunNabi** [comment]
After the typescript fixes were merged. This new error appeared during the test

```
ModSubmissionsEditFormFacility.vue:38 Message compilation error: Invalid linked format
1  |  e.g example@mail.com
```

for some reason `mail.com` or any .com I tried was not viable. So it wouldn't render the form due to that error.

**Comment by NabbeunNabi** [comment]
The only thing I would change now is go back into the cypress and put the translation strings from the `en.US` file back in the test

File: i18n/locales/en.json [hunk 1]
```diff
@@ -123,6 +123,53 @@
...
+        "labelFacilityGoogleMapsUrl": "Google maps URL",
+        "labelFacilityMapLatitude": "Map latitude",
+        "labelFacilityMapLongitude": "Map longitude",
+        "placeholderTextFacilityAddressLine1En": "Street address",
+        "placeholderTextFacilityAddressLine2En": "Apartment, building, room...",
+        "placeholderTextFacilityAddressLine1Jp": "Street address",
+        "placeholderTextFacilityAddressLine2Jp": "Apartment, building, room...",
+        "placeholderTextFacilityCityEn": "e.g. Shibuya",
+        "placeholderTextFacilityCityJp": "e.g. 渋谷区",
+        "placeholderTextFacilityEmail": "e.g example{'@'}mail.com",
```
Comment (NabbeunNabi) [inline i18n/locales/en.json:164]: WAHOOO!!! The error was found

**Review by NabbeunNabi — APPROVED** [review]

**Pull request merged**